examples/stable/mysql-csi-snapshot/README.md

```diff
@@ -0,0 +1,267 @@
+
+```bash
+$ kubectl create -f ./mysql-csi-snapshot-blueprint.yaml -n kanister
+```
+
+### Backup the application data
+
+Snapshot MySQL database using the backup ActionSet from above blueprint. Create an ActionSet in the `kanister` namespace. An easy way to do this is by using `kanctl`.
+
+```bash
+# Create Actionset
+# Please make sure the value of blueprint matches with the name of blueprint that was created earlier
+# --pvc argument specifies the PVC on which the action is performed
+$ kanctl create actionset --action backup --namespace kanister --blueprint mysql-csi-snapshot-bp --pvc mysql/$(kubectl get pvc -n mysql --selector=app.kubernetes.io/instance=mysql-release -o=jsonpath='{.items[0].metadata.name}')
+actionset backup-mlvcv created
+
+$ kubectl --namespace kanister get actionset
+NAME                         AGE
+backup-mlvcv                 112s
+
+# View the status of the actionset
+# Please make sure the name of the actionset here matches with name of the actionset created above
+$ kubectl --namespace kanister describe actionset backup-mlvcv
+
+# Check the CSI VolumeSnapshot created
+$ kubectl -n mysql get volumesnapshot
+NAME                                  READYTOUSE   SOURCEPVC              SOURCESNAPSHOTCONTENT   RESTORESIZE   SNAPSHOTCLASS      SNAPSHOTCONTENT                                    CREATIONTIME   AGE
+data-mysql-release-0-snapshot-cd26z   true         data-mysql-release-0                           8Gi           do-block-storage   snapcontent-2e411d1d-0b5f-48d4-9b79-b19e824c2e38   3h40m          3h40m
+```
+
+## Disaster strikes!
+
+Let's say someone accidentally deleted the test database.
+
+```bash
+# Connect to MySQL by running a shell inside MySQL's pod
+$ kubectl exec -it $(kubectl get pods -n mysql --selector=app.kubernetes.io/instance=mysql-release -o=jsonpath='{.items[0].metadata.name}') -n mysql -- bash
+
+$ mysql --user=root --password=asd#45@mysqlEXAMPLE
+
+mysql> SHOW DATABASES;
++--------------------+
+| Database           |
++--------------------+
+| information_schema |
+| my_database        |
+| mysql              |
+| performance_schema |
+| sys                |
+| test               |
++--------------------+
+6 rows in set (0.00 sec)
+
+mysql> DROP DATABASE test;
+Query OK, 1 row affected (0.12 sec)
+
+mysql> SHOW DATABASES;
++--------------------+
+| Database           |
++--------------------+
+| information_schema |
+| my_database        |
+| mysql              |
+| performance_schema |
+| sys                |
++--------------------+
+5 rows in set (0.01 sec)
+```
+
+## Restore MySQL DB
+
+To restore the missing data, you should use the backup that you created before. We will use `kanctl` for creating this restore action.
```

Comment:
```suggestion
To restore the missing data, use the backup created earlier. This can be achieved using `kanctl` for creating the restore action.
```
Noticed this inconsistency in all the other docs as well. Something to do in the future for others but let's start with this one. We should either not use `you, your etc.` or `we, us, our etc.` anywhere in the document or use it everywhere. I would recommend making the documentation neutral.
For example:
One of the sentences above says
`Create the Blueprint in the kanister namespace.` which is neutral instead of saying, `You can create a blueprint in the kanister namespace` which is directed towards the reader.